plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2729,7 +2729,7 @@ public boolean supportsReportingWrittenBytes(ConnectorSession session, SchemaTab
     private void cleanExtraOutputFiles(ConnectorSession session, String baseLocation, List<DataFileInfo> dataFiles)
     {
         Set<String> writtenFilePaths = dataFiles.stream()
```

Comment:
@losipiuk do you have any ideas how we could actually test the effectiveness of the cleaning method ?
we can't get the query id before executing the query.

I was thinking to create a file before hand, and rename it to match the cleaning criteria to verify after the query that it has been removed. Maybe event listener ?? It  seems though overly complicated...

Reply:
I think there is no easy way. Delta lake workers tend to clean after itself, so unless the worker dies there really nothing to clean.
Also it is not super big deal if we leave stuff behind.

Reply:
Also per @findinpath request I addressed that in: https://github.com/trinodb/trino/pull/17560